src/Runner/Parallel/ParallelConfigFactory.php

```diff
@@ -47,8 +49,11 @@ public static function detect(
             ]);
         }
 
+        // Reserve 1 core for the main orchestrating process
+        $available = self::$cpuDetector->getAvailableForParallelisation(1, $maxProcesses);
```

Comment:
is this really necessary? Once you stat the parallel processes, the main process is likely gonna sleep